portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,182 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
 }
 
 #greeting-container {
+  margin-top: 3px;
+  color:white;
```

Comment:
nit: Throughout this file, let's make sure there's a space between each CSS property name and the value.

Reply:
fixed